src/controllers/dashboard/plugins/installed/index.js

```diff
@@ -41,12 +55,18 @@ function getPluginCardHtml(plugin, pluginConfigurationPages) {
     })[0];
     const configPageUrl = configPage ? Dashboard.getPluginUrl(configPage.Name) : null;
     let html = '';
-    html += "<div data-id='" + plugin.Id + "' data-name='" + plugin.Name + "' data-removable='" + plugin.CanUninstall + "' class='card backdropCard'>";
+    html += "<div data-id='" + plugin.Id + "' data-version='" + plugin.Version + "' data-name='" + plugin.Name + "' data-removable='" + plugin.CanUninstall + "' data-status='" + plugin.Status + "' class='card backdropCard'>";
```

Comment:
Can you do this with backticks instead? https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/template_strings

Reply:
Been a long time absent from js. You mean like
```
html += "<div data-id='${plugin.Id}' data-version='${plugin.Version}' data-name='${plugin.Name}' data-removable='${plugin.CanUninstall}' data-status='${plugin.Status}' class='card backdropCard'>";
```

Reply:
@BaronGreenback Exactly like this:
```suggestion
    html += `<div data-id='${plugin.Id}' data-version='${plugin.Version}' data-name='${plugin.Name}' data-removable='${plugin.CanUninstall}' data-status='${plugin.Status}' class='card backdropCard'>`;
```